packages/admin/admin/src/form/FinalFormSelect.tsx

```diff
@@ -35,10 +35,7 @@ export const FinalFormSelect = <T,>({
     ...rest
 }: FinalFormSelectProps<T> & Partial<AsyncOptionsProps<T>> & Omit<SelectProps, "input">) => {
     const selectEndAdornment = clearable ? (
-        <>
-            <ClearInputAdornment position="end" hasClearableContent={Boolean(value)} onClick={() => onChange(undefined)} />
-            {endAdornment}
-        </>
+        <ClearInputAdornment position="end" hasClearableContent={Boolean(value)} onClick={() => onChange(undefined)} />
```

Comment:
This seems like a breaking change to me. Previously, it was possible to set `clearable` and an `endAdornment`, now it's not. Or am I mistaken?

Reply:
Yes, technically it's a breaking change. I doubt that any of our projects used `clearable` in combination with `endAdornment`, however. We could argue that it's okay to mark this as non-breaking.